Examples/Python/src/Vertexing.cpp

```diff
@@ -24,16 +24,25 @@ namespace py = pybind11;
 
 using namespace ActsExamples;
 using namespace Acts;
+using Seeder = ActsExamples::AdaptiveMultiVertexFinderAlgorithm::SeedFinder;
```

Comment:
nitpick: I think that `using` could be inside `addVertexing` as we do not need it anywhere else